src/toast/ops/noise_model.py

```diff
@@ -407,6 +424,16 @@ def _fit_log_jac(self, x, *args, **kwargs):
             f_alpha * log_freqs + fmin_alpha * np.log(fmin)
         ) / fmalpha
         return J
+    
+    def _get_err_ret(self, psd_unit):
```

Comment:
Could use a comment to explain what it does :)

Reply:
Added...